src/transform/ThemeTransform.css

```diff
@@ -35,17 +35,121 @@
      https://en.m.wikipedia.org/wiki/Wolverine_(character)?oldid=789987909.
      https://phabricator.wikimedia.org/diffusion/SMIN/browse/master/resources/skins.minerva.content.styles/hacks.less;2cc9516cde6027234de9881d503ed7161e0e2ace$44 */
   background: #27292d !important;
+  color: #f8f9fa !important;
 }
 
+.pagelib-theme-sepia .content table.wikitable th,
+.pagelib-theme-sepia .content table.mbox-small,
+.pagelib-theme-sepia .content table.mbox-small-left,
 .pagelib-theme-sepia .content table.infobox,
+.pagelib-theme-sepia .content table.infobox.vcard td.toccolours,
+.pagelib-theme-sepia .content table.infobox.vcard td div,
 .pagelib-theme-sepia .content table.infobox th {
   background: #f0e6d6 !important;
+  color: #222 !important; 
+}
+
+.pagelib-theme-dark .content table.tracklist th.tlheader.mbox-text {
+  background: #27292d !important; 
+} 
+
+.pagelib-theme-dark .content table.tracklist th.tlheader,
+.pagelib-theme-dark .content table.tracklist td span {
+  background: #27292d !important; 
+}
+
+.pagelib-theme-dark .content table.tracklist td {
+  background: #27292d !important; 
+}
+
+.pagelib-theme-sepia .content table.tracklist th.tlheader.mbox-text {
+  background: #f0e6d6 !important; 
+}
+
+.pagelib-theme-sepia .content table.tracklist th.tlheader,
+.pagelib-theme-sepia .content table.tracklist td span {
+  background: #f0e6d6 !important; 
+}
+
+.pagelib-theme-sepia .content table.tracklist td {
+  background: #f0e6d6 !important; 
+}
+
+.pagelib-theme-dark .content div.quotebox {
+  background: #27292d !important;
+}
+
+.pagelib-theme-sepia .content div.quotebox {
+  background: #f0e6d6 !important;
+}
+
+.pagelib-theme-dark .content table.mbox-small,
+.pagelib-theme-dark .content table.mbox-small-left,
+.pagelib-theme-dark .content table.wikitable,
+.pagelib-theme-dark .content table.wikitable tr th,
+.pagelib-theme-dark .content table.wikitable tr td,
+.pagelib-theme-dark .content div.quotebox,
+.pagelib-theme-dark .content div.noprint.portal.plainlist {
+  border-color: #43464a !important;
+}
+
+.pagelib-theme-sepia .content table.mbox-small,
+.pagelib-theme-sepia .content table.mbox-small-left,
+.pagelib-theme-sepia .content table.wikitable,
+.pagelib-theme-sepia .content table.wikitable tr th,
+.pagelib-theme-sepia .content table.wikitable tr td,
+.pagelib-theme-sepia .content div.quotebox,
+.pagelib-theme-sepia .content div.noprint.portal.plainlist {
+  border-color: #cbc8c1 !important;
+}
+
+.pagelib-theme-dark .content table.mbox-small-left hr,
+.pagelib-theme-dark .content table.mbox-small hr {
+  border-style: solid !important;
+  background: #43464a !important;
+  border-color: #43464a !important;
+}
+
+.pagelib-theme-sepia .content table.mbox-small-left hr,
+.pagelib-theme-sepia .content table.mbox-small hr {
+  border-style: solid !important;
+  background: #cbc8c1 !important;
+  border-color: #cbc8c1 !important;
+}
+
+.pagelib-theme-dark .content div.noprint.portal.plainlist > ul {
```

Comment:
Is it intentional to limit this rule to direct children of `div.noprint.portal.plainlist`?